### DISCUSSION TITLE
Inserted data via API not visible in Grid view [Jan 26]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5008] [Discord Threads]
> 🙋‍♂️ **flyomotive**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### Try refreshing table view. If that's not showing any data- then it's likely that insert API has failed. If you can share docker logs, that could help.  
#  
> 🙋‍♂️ **flyomotive**  
#### I assume you mean the reload button, right?  :)I have tried that, I also tried restarting the server and I created a separate table with the same rows and the exact same thing is happening there.  
#  
> 🙋‍♂️ **flyomotive**  
#### I'll try to get those logs. And thank you!  
#  
> 🙋‍♂️ **flyomotive**  
#### Here's an excerpt that I think might be relevant:   
#### ```   
#### Dec 23 18:06:50 GET /dashboard 301 185 - 18.799 ms   
#### Dec 23 18:06:50 GET /api/v1/db/meta/projects/p_q0x9vl95rsvukt/tables 200 1093 - 4.387 ms   
#### Dec 23 18:06:56 POST /api/v1/db/data/bulk/noco/p_q0x9vl95rsvukt/md_3pq27twif8zdn9 200 1129 - 98.965 ms   
#### Dec 23 18:07:00 GET /dashboard 301 185 - 4.683 ms   
#### Dec 23 18:07:10 GET /dashboard 301 185 - 0.840 ms   
#### Dec 23 18:07:15 GET /api/v1/db/data/noco/p_q0x9vl95rsvukt/md_3pq27twif8zdn9/views/vw_vh4cakgo0m463s?offset=0&limit=25 304 - - 9.899 ms   
#### Dec 23 18:07:16 GET /api/v1/db/data/noco/p_q0x9vl95rsvukt/md_3pq27twif8zdn9/views/vw_vh4cakgo0m463s?offset=0&limit=25 304 - - 8.940 ms   
#### Dec 23 18:07:20 GET /dashboard 301 185 - 0.818 ms   
#### Dec 23 18:07:27 GET /api/v1/db/data/noco/p_q0x9vl95rsvukt/md_3pq27twif8zdn9/views/vw_vh4cakgo0m463s?offset=0&limit=25 304 - - 7.168 ms   
#### Dec 23 18:07:30 GET /dashboard 301 185 - 0.847 ms   
#### Dec 23 18:07:33 GET /api/v1/db/meta/views/vw_vh4cakgo0m463s/filters 304 - - 2.305 ms   
#### Dec 23 18:07:39 GET /api/v1/db/data/noco/p_q0x9vl95rsvukt/md_3pq27twif8zdn9?fields=link&limit=100&offset=0 200 99 - 5.247 ms   
#### Dec 23 18:07:40 GET /dashboard 301 185 - 6.458 ms   
#### ```  
#  
> 🙋‍♂️ **flyomotive**  
#### At 18:06:56 I inserted 94 records into the table and the response code was 200   
#### At 18:07:39 I queried the table and got nothing in reponse, but the code was also 22   
#### When I try to access the dashboard, it logs `GET /dashboard` with a 301 and also a bunch of 304s when I reload the page.  
#  
> 🙋‍♂️ **flyomotive**  
#### Is there any way to get more verbose logs to see what's going on under the hood?  
#  
> 🙋‍♂️ **flyomotive**  
#### Googling for "NocoDB 301/304" gets me nowhere, I don't exactly see what 301 means in this context either.  
#  
> 🙋‍♂️ **flyomotive**  
#### Now the record count is accurate for some reason (I changed nothing) but there still is no display  
![image0](https://cdn.discordapp.com/attachments/1055608787324649602/1055896655439855666/image.png)  
#  
> 🙋‍♂️ **flyomotive**  
#### Ahh, it's back to 0  
#  
> 🙋‍♂️ **flyomotive**  
#### Whenever I create a record by hand, the count displays the right number. But when I refresh, the hand-created record is gone and the count is back to 0.  
#  
> 🙋‍♂️ **semighoti**  
#### check your n8n setting  
#  
> 🙋‍♂️ **semighoti**  
#### did you choose the correct Table Name or ID on n8n?  
#  
> 🙋‍♂️ **semighoti**  
#### I have no issue with the importing data from n8n to nocodb  
#  
> 👨‍💻 **pranavxc**  
#### Is there any formula or filter ?  
#  
> 🙋‍♂️ **semighoti**  
#### Not really, but the columns' name on both n8n and nocodb has to be matched  
#  
> 👨‍💻 **pranavxc**  
#### <@748635187239190600> sorry the question is for  <@243865621983526912> 🙂   
####    
#### <@243865621983526912> - Is there any formula or filter ?  
#  
> 🙋‍♂️ **flyomotive**  
#### yeah, and the inserts show up in the audit log. when I use the exact same settings to GET some records, nothing is returned. but i will check again 🫡  
#  
> 🙋‍♂️ **flyomotive**  
#### Actually I played around with formula filters so there might be something active, I will check as soon as I can.  
#  
> 🙋‍♂️ **flyomotive**  
#### There is no Filter applied right now, but maybe I just can't find it?  
#  
> 🙋‍♂️ **flyomotive**  
#### If you watn you can take a look at the table here: https://db.lgnt.eu/dashboard/#/nc/view/79347319-e95c-4e26-ac88-6867692facea  
#  
> 🙋‍♂️ **flyomotive**  
#### The password is ```8bUz7ePjwosi5A```  
#  
> 🙋‍♂️ **flyomotive**  
#### You can also try adding rows and see what happens on your end  
#  
> 🙋‍♂️ **flyomotive**  
#### When I add rows, I get  
![image0](https://cdn.discordapp.com/attachments/1055608787324649602/1056987884315480206/image.png)  
#  
> 🙋‍♂️ **flyomotive**  
#### And that's the corresponding Log:  
![image0](https://cdn.discordapp.com/attachments/1055608787324649602/1056988119796293723/image.png)  
#  
> 🙋‍♂️ **flyomotive**  
#### Something wrong with the Id column, which would explain why the filed "id" always says "none" in the response of my POST calls.  
#  
> 🙋‍♂️ **flyomotive**  
#### I don't know why that happens though since I have set (like the default setting is) Id column to auto-increment  
#  
> 🙋‍♂️ **flyomotive**  
#### And it does show it briefly with a correct auto-incremented numerical value (2000-something in my case) before it disappears again like I described above.  
#  
> 🙋‍♂️ **flyomotive**  
#### I think my best bet to fix this is to reinstall NocoDB completely and use an external database instead of the internal SQLite  
#  
> 🙋‍♂️ **flyomotive**  
#### Don't know how to install one though since I'm on Cloudron  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### <@243865621983526912> are you trying to add records from shared view link?  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### link shared above is shared view link. Shared views are read-only  
#  
> 🙋‍♂️ **Dstala | NocoDB Committer**  
#### https://db.lgnt.eu/dashboard/#/nc/view/79347319-e95c-4e26-ac88-6867692facea  
#  
> 👨‍💻 **pranavxc**  
#### <@243865621983526912> - it's trying to access some meta table id and not the data table. Can you share the nocodb app version you are using.  
#  
> 🙋‍♂️ **flyomotive**  
#### I tried it once, but I encountered all my issues before that.  
#  
> 🙋‍♂️ **flyomotive**  
#### Interesting! My Cloudron tells me I'm on NocoDB 0.100.2  
#  
> 👨‍💻 **pranavxc**  
#### let me check production source code  
#  
> 👨‍💻 **pranavxc**  
#### <@243865621983526912>  Please check your DM, I would like to know more details  
#  
> 🙋‍♂️ **flyomotive**  
#### Sure thing. And thank you lots!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1055608787324649602)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
